app/assets/stylesheets/components/messages.scss

```diff
@@ -0,0 +1,12 @@
+@import 'govuk-frontend/dist/govuk/helpers/colour';
+
+// Message header styling for sender messages (blue) and recipient messages (grey)
+.message-header--blue {
+  background-color: govuk-colour('blue') !important;
+  color: govuk-colour('white') !important;
+}
+
+.message-header--grey {
```

Comment:
IMHO it would be better to name this what it is (recipient_message) rather than message-header-grey

Reply:
I don't think recipient_message is what this is. Every message is both a sender_message and a recipient_message as messages by their nature always have a sender and recipient. I think this name better describes what the style is.